manifests/istio-control/istio-discovery/templates/telemetryv2_1.5.yaml

```diff
@@ -144,6 +196,103 @@ spec:
                     local:
                       inline_string: envoy.wasm.stats
 ---
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: tcp-stats-filter-1.5
```

Comment:
nit/question: do we actually want this to be a separate envoyfilter from http? I think internally we merge anyways so there isn't perf issues either way

Reply:
this is added to network filter chain for tcp and the other one to HCM